tests/utils_test.py

```diff
@@ -0,0 +1,22 @@
+"""Unit tests for english language checker"""
+import unittest
+import libcipher.utils as utils
+
+filepath = '/home/travis/build/Chennaipy/libcipher/libcipher/dictionary.txt'
```

Comment:
@Anupama-Github how about changing the API to accept a file object instead of a filename? This will make the API more flexible, and in the test case we could pass in a StringIO object, with the dictionary contents.